### PR TITLE
Add laravel 6 support

### PR DESCRIPTION
**Pull request by Okipa**
Add Laravel 6 support.

Related issue : https://github.com/Edujugon/laradoo/issues/17.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,9 +20,10 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "illuminate/support": "^5.1"
+        "php": "^5.6|^7.0",
```
Comment (Edujugon) [inline composer.json]: Could you please replace the single pipes with doubles?
Based on the [documentation](https://getcomposer.org/doc/articles/versions.md#version-range), that's the recommended "or" operator

File: readme.md [hunk 2]
```diff
@@ -9,6 +9,12 @@ Odoo ERP API for Laravel. [Odoo website](https://www.odoo.com)
 [![Latest Stable Version](https://poser.pugx.org/edujugon/laradoo/v/stable)](https://packagist.org/packages/edujugon/laradoo)
 [![License](https://poser.pugx.org/edujugon/laradoo/license)](https://packagist.org/packages/edujugon/laradoo)
 
+## Compatibility
+
+| Laravel version | PHP version | Package version |
+|---|---|---|
+| ^5.1 | ^5.5 | ^V1.1 |
+| ^5.1 | ^5.6 | ^V1.2 |
```
Comment (Edujugon) [inline readme.md]: Since there is a possibility that this PR becomes a breaking change for someone, I would rather release the V2 instead of v1.2

**Comment by Okipa** [comment]
Thanks for your review @Edujugon, I did the requested changes, it should be ok now.

**Comment by Edujugon** [comment]
@Okipa ,

Thanks for your contribution 🎉

**Pull request merged**